### PR TITLE
[SCRUM-70] Recsys model optimization file upload

### LINKED ISSUE #65: ✨ feat: RecSys Model Optimization
# Title

- 추천시스템 모델 최적화 Research

# TODO

- [x] Kakao_Recommend_System(Research_Model_Optimization).ipynb

# etc

- 앙상블 모델 사용 결정.

### PR DESCRIPTION
**Pull request by Daehyun-Bigbread**
# ☝️Issue Number

- #65 

# 🔎 Key Changes
- 평가지표 결과 Research

# 💌 To Reviewers
- 앙상블 모델 사용 결정.

평가지표 점수:
                         MSE       MAE      RMSE  R-squared      MAPE
Linear Regression  0.189934  0.397975  0.435815  -0.192038  0.093079
Ridge Regression   0.189507  0.397458  0.435323  -0.189352  0.092966
Lasso Regression   0.160264  0.356250  0.400330  -0.005826  0.083846
Random Forest      0.207377  0.347250  0.455386  -0.301507  0.083079
Gradient Boosting  0.274372  0.397990  0.523806  -0.721974  0.095526
XGBoost            0.244682  0.328111  0.494653  -0.535636  0.078643
LightGBM           0.209661  0.407458  0.457888  -0.315845  0.095972
SVM                0.179939  0.315760  0.424192  -0.129305  0.076260
KNN                0.186113  0.332500  0.431408  -0.168051  0.080110
Ensemble (Avg)     0.012113  0.072404  0.110060   0.898825  0.016432

평가지표별 순위:
                    MSE Rank  MAE Rank  RMSE Rank  R-squared Rank  MAPE Rank
Linear Regression       6.0       8.0        6.0             6.0        8.0
Ridge Regression        5.0       7.0        5.0             5.0        7.0
Lasso Regression        2.0       6.0        2.0             2.0        6.0
Random Forest           7.0       5.0        7.0             7.0        5.0
Gradient Boosting      10.0       9.0       10.0            10.0        9.0
XGBoost                 9.0       3.0        9.0             9.0        3.0
LightGBM                8.0      10.0        8.0             8.0       10.0
SVM                     3.0       2.0        3.0             3.0        2.0
KNN                     4.0       4.0        4.0             4.0        4.0
Ensemble (Avg)          1.0       1.0        1.0             1.0        1.0
...
KNN                       24.0           4.0
Ensemble (Avg)             6.0           1.0

가장 성능이 좋은 모델은: Ensemble (Avg)

### CODE REVIEW

**Comment by Daehyun-Bigbread** [comment]
모듈화 및 코드 수정은 내일 진행 예정.
